test/unit/esm-querystring/input.js

```diff
@@ -0,0 +1,5 @@
+// Test that querystrings of various forms get stripped from esm imports
```

Comment:
Lets add `package.json` to this dir with `"type": "module"`, otherwise its wont be valid for Node.js